source/particles/particle_handler.cc

```diff
@@ -595,35 +597,71 @@ namespace Particles
     std::map<unsigned int, std::vector<std::vector<double>>>
       locally_owned_properties_from_other_processes;
 
-    if (n_global_properties > 0)
+    // A map from mpi process to ids, ordered as in the IndexSet.
+    // Notice that this ordering maybe different from the ordering in the
+    // vectors above, since no local ordering is guaranteed by the
+    // distribute_compute_point_locations() call.
+    // This is only filled if ids.size() is > 0
```

Comment:
I might have already forgotten, but did you say that in the documentation?

Reply:
yes. :)